src/WebJobs.Script/Content/Script/functions.js

```diff
@@ -33,11 +33,17 @@ function createFunction(f) {
 
         f = getEntryPoint(f, context);
 
-        var origLog = context.log;
-        context.log = function () {
-            value = util.format.apply(null, arguments);
-            origLog(value);
-        };
+        var loggers = context.log;
```

Comment:
I'm finding the following code very difficult to understand/read. Maybe its just me, but its making my head hurt